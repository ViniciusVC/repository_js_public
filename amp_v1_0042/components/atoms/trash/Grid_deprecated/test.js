import React from "react";
import { shallow } from "enzyme";
import Component from "./index";

describe("Grid >", () => {
  it("deve renderizar com o tema default", () => {
    const wrapper = shallow(<Component />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve renderizar com um container de 1023px", () => {
    const wrapper = shallow(<Component container="1023px" />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve aceitar um tema", () => {
    const wrapper = shallow(
      <Component
        theme={{
          flex: `
    position: relative;
    display: flex !important;
    `
        }}
      />
    );
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
});
