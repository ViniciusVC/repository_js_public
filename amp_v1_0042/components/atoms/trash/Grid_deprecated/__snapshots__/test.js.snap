// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid > deve aceitar um tema 1`] = `
<main
  className="grid sc-bdVaJa hlxBJy"
/>
`;

exports[`Grid > deve renderizar com o tema default 1`] = `
<main
  className="grid sc-bdVaJa dQMYgd"
/>
`;

exports[`Grid > deve renderizar com um container de 1023px 1`] = `
<main
  className="grid sc-bdVaJa dXIQpP"
/>
`;
